docs/readmes/lte/ha_setup.md

```diff
@@ -0,0 +1,179 @@
+---
+id: ha_setup
+title: HA AGW Setup
+sidebar_label: HA AGW Setup
+hide_title: true
+---
+# HA AGW Setup
+
+The following document outlines the necessary steps to deploy and configure a
+Magma access gateway on AWS. This document also outlines configuring the AWS
+gateway to serve as a secondary to a primary gateway running at an edge site.
```

Comment:
Do we have a HA AGW feature document to provide more information in case someone is curious to learn more about this ?